jumpstarter/states.py

```diff
@@ -19,17 +19,33 @@ class ActorStoppingState(Enum):
     dependencies_stopped = auto()
 
 
+class ActorRestartingState(Enum):
+    starting = auto()
+    stopping = auto()
+
+
 class ActorState(Enum):
     initializing = auto()
     initialized = auto()
-    # restarting = ActorRestartState
     starting = ActorStartingState
     started = auto()
     stopping = ActorStoppingState
     stopped = auto()
+    restarting = ActorRestartingState
     crashed = auto()
 
 
+class ActorStateTriggers(str, Enum):
+    def _generate_next_value_(name, *args):
+        return name.lower()
+
+    init = auto()
```

Comment:
```suggestion
    initialize = auto()
```

I prefer to be clearer even if we're more verbose as we have auto-completion.